lib/assertions.js

```diff
@@ -390,7 +386,7 @@ module.exports = function (expect) {
                             // No common prefix, omit diff
                             return null;
                         } else {
-                            output.partialMatch(subject.substr(0, i)).text(subject.substr(i));
+                            output.match(subject.substr(0, i)).text(subject.substr(i), true);
```

Comment:
That's maybe sort of an argument for reintroducing the `partialMatch` style -- clarity.